packages/ui/src/components/va-date-input/VaDateInput.vue

```diff
@@ -3,7 +3,9 @@
     <va-dropdown
       v-model="isOpenSync"
       :offset="[2, 0]"
-      :close-on-content-click="false"
+      :close-on-content-click=false
```

Comment:
Why no quotes here and below?

Reply:
> Because passing it with quotes will pass a `string` instead of expected `boolean`

Isn't `close-on-content-click="false"` will pass string, but `:close-on-content-click="false"` boolean? Looks at syntax highlight:

`:close-on-content-click=false` looks weird for me.

Reply:
> `:` forces Vue not to accept value as string but as js expression.
> 
> :attr="false" - boolean. attr="false" - string.
> 
> https://vuejs.org/guide/components/props.html#boolean

Yes, I thought that this is a more explicit way to assign `boolean`.
Because in pure JS expression like this - "false" will return a `string`.

Reply:
> > Because passing it with quotes will pass a `string` instead of expected `boolean`
> 
> Isn't `close-on-content-click="false"` will pass string, but `:close-on-content-click="false"` boolean? Looks at syntax highlight:
> 
> `:close-on-content-click=false` looks weird for me.

That's right.

By the way quotes here are completely optional - same goes to all `html5` `attributes` as well.

For me it looked more clear to assign `boolean` this way,
Though we could have quotes everywhere just for consistency because they will be removed while minified anyway.